test_driver/utils/common_actions.dart

```diff
@@ -8,6 +8,8 @@ import 'localizations.dart';
 
 class Keys {
   static const String openAppDrawer = 'Open navigation menu';
+  static const String drawerKey = 'drawer_key';
```

Comment:
In general please try to create as few of these keys as possible, I think it's much better if the tests are mapped to the text in the app. I realize sometimes a key is needed though...